frontend/src/app/data-entry/agent-form/agent-form.component.html

```diff
@@ -0,0 +1,21 @@
+<ng-container *ngIf="data$ | async as data">
+    <lc-breadcrumb [breadcrumbs]="getBreadcrumbs(data)"></lc-breadcrumb>
```

Comment:
In my personal opinion, self-closing tags look just a bit neater if there are no slots involved.

```suggestion
    <lc-breadcrumb [breadcrumbs]="getBreadcrumbs(data)" />
```